tests/integration/modules/test_state.py

```diff
@@ -1913,6 +1994,8 @@ def test_state_sls_id_test(self):
         testfile = os.path.join(TMP, 'testfile')
         comment = 'The file {0} is set to be changed'.format(testfile)
         ret = self.run_function('state.sls', ['core'])
+        log.error("HERE")
```

Comment:
@mattp- Looks like there is some leftover debug cruft here.

Reply:
actually put that there on purpose :) windows keeps failing this in jenkins and I don't know why